pkg/processor/procbuiltin/unwrap.go

```diff
@@ -93,6 +96,78 @@ func (p *unwrapProcessor) Process(_ context.Context, in record.Record) (record.R
 	return out, nil
 }
 
+/*
+Example of an OpenCDC record:
+{}
+*/
+
+// openCDCUnwrapper unwraps an OpenCDC record from the payload, by unmarhsalling rec.Payload.After into type Record.
+type openCDCUnwrapper struct{}
+
+func (o *openCDCUnwrapper) Unwrap(rec record.Record) (record.Record, error) {
```

Comment:
the goal in this method is to serialize the entire message in kafka to `record.Record` struct type